INDEX.HTML

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html style=" background-color:aqua;
+text-align:center;
+" >
+    <head>
+    
+    <link rel="stylesheet" href="project-1.css">
+    
+    </head>
+    <body>
+    <div class="mid" >
```

Comment:
please, use the Meaningful Tags In Html Code.
Using the Semantic HTML Elements, because it makes our code much easier to read.

Reply:
[OPTIONAL] 👍🏽 
- Great Job stating WHY  a change should be made 🤗 . I think you can reconsider changing this statement to be polite by saying

"I think it will be better to use meaningful Tags in HTML codes because it makes our code much easier to read. 👍🏽. Please kindly consider using semantic HTML 😄 "

_What do you think about this comment I have just made above?? It is polite, friendly, gives reason WHY  a change should be made 👍🏽_